### PR TITLE
Bump elliptic version

### PR DESCRIPTION
**Pull request by jason-upchurch**
### Summary
replaces dependabot PR #516 

### How to test:
@rfultz I bumped elliptic and added as a devDependency
@fec-jli @pkfec I was able to run local api and parse locally

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
   },
   "devDependencies": {
     "browserify": "^13.0.0",
+    "elliptic": "^6.5.3",
```
Comment (pkfec) [inline package.json:12]: Any good reason you put this pkg in package.json file?

Comment (jason-upchurch) [reply package.json:12]: yes, but I am open to other suggestions.

Comment (rfultz) [reply package.json:12]: Yeah, it's pretty common to add a package here and then let this version satisfy where else it's a dependency

**Review by pkfec — APPROVED** [review]
Looks good @jason-upchurch

**Review by fec-jli — APPROVED** [review]
work as expected. Thanks

**Pull request merged**